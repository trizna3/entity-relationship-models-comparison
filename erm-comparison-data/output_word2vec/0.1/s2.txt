hracie plochy-hracie plochy
hry-hry
pocasia-pocasia
typy pohonov-pohon
jednotky-jednotky
policko-pocasie na ploche
typy policok-typ
REBIND_MN_TO_1NN1:S-hracie plochy-pocasia
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-typ-pohon
