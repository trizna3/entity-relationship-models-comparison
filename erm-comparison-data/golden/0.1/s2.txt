hry-hry
hracie plochy-hracie plochy
policko-policka
typy policok-typ
pocasia-pocasia
typy pohonov-pohon
jednotky-jednotky
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-typ-typy policok